src/spec/text-spec/font-spec.ts

```diff
@@ -1,9 +1,17 @@
+import { TFontFamily } from "./types";
+
 export const FontFamily = {
     OpenSans: {
         Regular: "Open Sans",
         Semibold: "Open Sans Semibold",
         Bold: "Open Sans Bold",
         Light: "Open Sans Light",
     },
+    PlusJakartaSans: {
+        Regular: "Plus Jakarta Sans",
+        Semibold: "Plus Jakarta Sans Semibold",
```

Comment:
noticed the uploaded file is using `SemiBold` and not `Semibold`, which one are we using?

- https://assets.life.gov.sg/react-design-system/fonts/PlusJakartaSans-Semibold.ttf
- https://assets.life.gov.sg/react-design-system/fonts/PlusJakartaSans-SemiBold.ttf

Reply:
we should be using Semibold to follow convention . Will update app-assets to change to `Semibold`

Thanks for catching

Reply:
Updated app assets to use `Semibold`